webpack.config.js

```diff
@@ -0,0 +1,41 @@
+const path = require('path');
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ] I would Kindly suggest removing this line and not disabling the linters because their purpose is to help you write better and clean code. 👌

Reply:
@ridwanediallo Thank you 💯 It is fixed.